course/include/Matrix3.h

```diff
@@ -0,0 +1,61 @@
+#pragma once
+#include <iostream>
+#include <cmath>
+#include "Vector3.h"
+
+namespace ekumen
+{
+
+    class Matrix3
+    {
+    public:
+        Matrix3();
+        explicit Matrix3(const Vector3 &f1, const Vector3 &f2, const Vector3 &f3);
+        Matrix3(const std::initializer_list<double> &list);
+        Matrix3(const std::initializer_list<double> &r1, const std::initializer_list<double> &r2, const std::initializer_list<double> &r3);
+        Matrix3(const Matrix3 &m);
+
```

Comment:
Add a move constructor. Since your vectors already implement movement, it can be as simple as:
```
Matrix3(Matrix3 &&m) {
  row_values[0] = std::move(m.row_values[0]);
  row_values[1] = std::move(m.row_values[1]);
  row_values[2] = std::move(m.row_values[2]);
}
```

Reply:
Done